frontend/src/pages/Landing/index.js

```diff
@@ -0,0 +1,331 @@
+      startDate,
+      author,
+      topics,
+      collaborators,
+      lastSaved,
+      status,
+      regionId,
+    } = report;
+
+    const recipientsTitle = activityRecipients.reduce(
+      (result, ar) => `${result + (ar.grant ? ar.grant.grantee.name : ar.name)}\n`,
+      '',
+    );
+
+    const recipients = activityRecipients.map((ar) => (
+      <Tag
+        key={`${ar.name.slice(1, 3)}_${ar.id}`}
+        className="smart-hub--table-collection"
+      >
+        {ar.grant ? ar.grant.grantee.name : ar.name}
+      </Tag>
+    ));
+
+    const topicsTitle = topics.reduce(
+      (result, topic) => `${result + topic}\n`,
+      '',
+    );
+
+    const topicsWithTags = topics.map((topic) => (
+      <Tag
+        key={`${topic.slice(1, 13)}`}
+        className="smart-hub--table-collection"
+      >
+        {topic}
+      </Tag>
+    ));
+
+    const collaboratorsTitle = collaborators.reduce(
+      (result, collaborator) => `${result + collaborator.fullName}\n`,
+      '',
+    );
+
+    const collaboratorsWithTags = collaborators.map((collaborator) => (
+      <Tag
+        key={`${collaborator.fullName.slice(1, 13)}`}
+        className="smart-hub--table-collection"
+      >
+        {collaborator.fullName}
+      </Tag>
+    ));
+
+    const fullId = !id
+      ? ''
+      : activityReportId(id, regionId);
+
+    return (
+      <tr key={`landing_${id}`}>
+        <td>
+          <Link
+            to={`/activity-reports/${id}/activity-summary`}
+            href={`/activity-reports/${id}/activity-summary`}
+          >
+            {fullId}
+          </Link>
+        </td>
+        <td>
+          <span className="smart-hub--ellipsis" title={recipientsTitle}>
+            {recipients}
+          </span>
+        </td>
+        <td>{startDate}</td>
+        <td>
+          <span className="smart-hub--ellipsis" title={author.fullName}>
+            {author.fullName}
+          </span>
+        </td>
+        <td>
+          <span className="smart-hub--ellipsis" title={topicsTitle}>
+            {topicsWithTags}
+          </span>
+        </td>
+        <td>
+          <span className="smart-hub--ellipsis" title={collaboratorsTitle}>
+            {collaboratorsWithTags}
+          </span>
+        </td>
+        <td>{lastSaved}</td>
+        <td>
+          <Tag
+            className={`smart-hub--table-tag-status smart-hub--status-${status}`}
+          >
+            {status}
+          </Tag>
+        </td>
+        <td>
+          <button type="button" className="smart-hub--dotdotdot">
+            ...
+          </button>
+        </td>
+      </tr>
+    );
+  });
+}
+
+function Landing() {
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [reports, updateReports] = useState([]);
+  const [error, updateError] = useState();
+  const [sortConfig, setSortConfig] = React.useState({
+    key: 'lastSaved',
+    direction: 'descending',
+  });
+
+  const sortedReports = [...reports];
+  const requestSort = (key) => {
```

Comment:
We'll probably need to move this sort to the API once we start paginating on the server